go/fn/const.go

```diff
@@ -41,3 +41,18 @@ const (
 	// KptLocalConfig marks a KRM resource to be skipped from deploying to the cluster via `kpt live apply`.
 	KptLocalConfig = ConfigPrefix + "local-config"
 )
+
+// For Kpt use only constants
+const (
+	// KptUseOnlyPrefix is the prefix of kpt-only annotations. Users are not expected to touch these annotations.
+	KptUseOnlyPrefix = "internal.kpt.dev/"
+
+	// UpstreamIdentifier is the annotation to record a resource's upstream origin.
+	// It is in the form of <GROUP>|<KIND>|<NAMESPACE>|<NAME>
+	UpstreamIdentifier = KptUseOnlyPrefix + "upstream-identifier"
+
+	// UnkownNamespace is the special char for cluster-scoped or unknown-scoped resources. This is only used in upstream-identifier
```

Comment:
> the special char

It's not a single character. It's more accurate to call it a special string. `~C` doesn't seems so meaningful and special. I'd suggest using something like `~UNKNOWN~`.

Reply:
> `~C` doesn't seems so meaningful and special. I'd suggest using something like `~UNKNOWN~`.

Yeah, that's true. 😞  Some more context: 
- This represents both cluster-scoped and unknown resources.
- Neither `Unknown/U` or `Cluster/C` give helpful info to explain this field. 
- Kustomize uses "__nonenamespacable__", which is too long.    

So we are suggested to use "~C"
https://docs.google.com/document/d/1YwxalwS45V1BhvzwWrgHJQPL2mZ1xchIzTm_eI6jXsk/edit?disco=AAAAbLmue7Y